app.js

```diff
@@ -73,7 +73,7 @@ async function authorizationMiddleware(req, res, next) {
   }
 
   if (authorization.startsWith('Basic')) {
-    if (process.env.NODE_ENV === 'test') {
+    if (process.env.NODE_ENV === 'test' || process.env.HRM_PERMIT_BASIC_AUTHENTICATIO) {
```

Comment:
Is there a reason (like a 30-character limit) to leave off the `N` in `AUTHENTICATION`?

Reply:
No, I renamed the variable at the last minute to be more inline with existing stuff, must've introduced a typo